### PR TITLE
Update about-forks.md

### PR DESCRIPTION
**Pull request by mnaderian**
Did not really understand the sentence where I made changes in so I thought maybe it was mistyped.

### CODE REVIEW

File: content/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ In open source projects, forks are often used to iterate on ideas or changes bef
 
 {% data reusables.repositories.private_forks_inherit_permissions %}
 
-If you want to create a new repository from the contents of an existing repository but don't want to merge your changes upstream in the future, you can duplicate the repository or, if the repository is a template, use the repository as a template. For more information, see "[Duplicating a repository](/articles/duplicating-a-repository)" and "[Creating a repository from a template](/articles/creating-a-repository-from-a-template)".
+If you want to create a new repository from the contents of an existing repository but don't want to merge your changes to the upstream in the future, you can duplicate the repository or, if the repository is a template, use the repository as a template. For more information, see "[Duplicating a repository](/articles/duplicating-a-repository)" and "[Creating a repository from a template](/articles/creating-a-repository-from-a-template)".
```
Comment (import-brain) [inline content/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks.md:38]:
```suggestion
If you want to create a new repository from the contents of an existing repository but don't want to merge your changes to the upstream in the future, you can duplicate the repository or, if the repository is a template, you can use the repository as a template. For more information, see "[Duplicating a repository](/articles/duplicating-a-repository)" and "[Creating a repository from a template](/articles/creating-a-repository-from-a-template)".
```
IMHO, this sentence is just worded weirdly, but adding this change would make it even better.

Comment (mnaderian) [reply content/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks.md:38]: Nice suggestion.

**Comment by import-brain** [comment]
LGTM! This repo doesn't let me send an approving review, weird...

**Review by ramyaparimi — APPROVED** [review]
@mnaderian 
Thanks so much for opening a PR with this fix! I'll update the branch and get this merged once tests are passing 🍏

**Pull request merged**